### ISSUE TITLE
Development? New version?

### ISSUE THREAD

**Issue by Neustradamus** [HewlettPackard/wireless-tools#9]
Last stable version "29" is from 2007-09-18
30pre9 : 2009-11-24

Any news?

**Comment by minanagehsalalma** [comment]
@Neustradamus I don't think so mate
.
I have an question 
Is it possible to get other APs channel's while broadcasting an ap and deauthing on a single adapter at the same time? Using iwlist ?

**Comment by Neustradamus** [comment]
@jean2: What do you think?

**Comment by Neustradamus** [comment]
@jean2: It is possible to add the wireless_tools.30.pre9 in the master and in Releases and Tags sections ?
- https://github.com/HewlettPackard/wireless-tools/releases

Thanks in advance.

**Comment by porteusconf** [comment]
TL/DR --- you are best off looking at source from debian, where newest as of 2024 is at [https://packages.debian.org/trixie/wireless-tools](https://packages.debian.org/trixie/wireless-tools)  They have some patches including a gcc9 patch to some Makefiles...  Near as I can tell, they start with the 30pre9 tarball then patch it:
[https://salsa.debian.org/debian/wireless-tools/-/tree/master?ref_type=heads](https://salsa.debian.org/debian/wireless-tools/-/tree/master?ref_type=heads)

 --- discussion ---
This HP  GitHub repo seems abandoned since at least 2017. There are 5 PRs that were never merged, even ones just correcting spelling/punctuation like this April 2020 commit ready to merge just the manpage wireless_tools/iwconfig.8 (only cosmetic changes, no changes to meaning or information  in the manpage):
[https://github.com/mb720/wireless-tools/tree/patch-1](https://github.com/mb720/wireless-tools/tree/patch-1)

mb720 did a PR in 2020 at [https://github.com/HewlettPackard/wireless-tools/pull/11](https://github.com/HewlettPackard/wireless-tools/pull/11) 

So I don't hold out hope for this repo to host the code from "30pre9" tarball at  [https://hewlettpackard.github.io/wireless-tools/wireless_tools.30.pre9.tar.gz](https://hewlettpackard.github.io/wireless-tools/wireless_tools.30.pre9.tar.gz) 

Theoretically, it may be possible to try to re-create the patches that resulted in the beta.  But since this repo seems abandoned, it may be better to find out where the debian maintainers host the source code:
[https://packages.debian.org/source/bookworm/wireless-tools](https://packages.debian.org/source/bookworm/wireless-tools)
Currently, May 2024, they still start with the same 348477 byte tarball: [http://deb.debian.org/debian/pool/main/w/wireless-tools/wireless-tools_30~pre9.orig.tar.gz](http://deb.debian.org/debian/pool/main/w/wireless-tools/wireless-tools_30~pre9.orig.tar.gz)  
The debian-source currently lists the most recent as 30~pre9-17, 30~pre9-14 and 30~pre9-13.1 per the changelog in [http://deb.debian.org/debian/pool/main/w/wireless-tools/wireless-tools_30~pre9-17.debian.tar.xz](http://deb.debian.org/debian/pool/main/w/wireless-tools/wireless-tools_30~pre9-17.debian.tar.xz)

While looking for repo where debian hosts their source, I see that  in 2017, dkg submitted a bug report where he offered to put debian source under VCS in a git repo, but I have not found such a repo yet... [https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=864658](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=864658)

**Comment by porteusconf** [comment]
[https://unix.stackexchange.com/questions/407517/why-did-wireless-tools-version-30-become-a-permanent-beta](https://unix.stackexchange.com/questions/407517/why-did-wireless-tools-version-30-become-a-permanent-beta) explains  quite well in some detail why network-tools with iwconfig seems abandoned:
> "Wireless-tools is deprecated in favor of iw because the wireless extensions has been deprecated in favor of the new nl80211 interface for wireless devices."
